app/app.rb

```diff
@@ -121,10 +121,11 @@
 
 post '/user/routes' do
   user = user_from_token
-  route = Route.create!(user: , commitment: user.current_commitment, started_at: Time.current)
 
-  { data: route.to_json }
+  route = Route.create!(user: , commitment: user.current_commitment, started_at: Time.current)
```

Comment:
What if the User doesn't have a current_commitment? Will this raise an error? Can we count on them always having one?

Reply:
They don't start out with a commitment so this should probably be changed to be optional.